### DISCUSSION TITLE
Redis sentinel read/write

### DISCUSSION THREAD

**Discussion by fjmoreno** [redis/rueidis#429] [Q&A] [answered]
Hi,
In a redis sentinel client, is there any way to send read operations to the slaves nodes and write operations to the master node?

Thank you in advance

**Comment by rueian** [comment, marked as answer]
Hi @fjmoreno,

 Currently, you will need two rueidis clients. One with the `ReplicaOnly` option set.

**Reply by fjmoreno** [reply]
ok, I realized that I was reading old documentation in which this option did not appear. Ill try that, thank you
